scripts/extras/update_task.apt

```diff
@@ -52,36 +52,39 @@ fi
 
 
 # Now run Foglamp update commands.
-
-UPDATE_CMD="sudo apt update"
-FOGLAMP_UPDATE_CMD="sudo apt-get -y install foglamp"
+UPDATE_CMD="sudo apt-get -y --allow-unauthenticated update"
+FOGLAMP_UPDATE_CMD="sudo apt-get -y --allow-unauthenticated install foglamp"
```

Comment:
All our foglamp packages which we get from http://archives.dianomic.com/ are gpg signed. We should not be passing this arg ```--allow-unauthenticated```

Reply:
I agree. Since I was testing with local repo, this was inducted to allow testing the PR per below documentation:
>--allow-unauthenticated
>Ignore if packages can't be authenticated and don't prompt about it. This can 
>be useful while working with local repositories, but is a huge security risk if data 
>authenticity isn't ensured in another way by the user itself. The usage of the Trusted 
>option for sources.list(5) entries should usually be preferred over this global override.
>Configuration Item: APT::Get::AllowUnauthenticated.

So I have removed it but please DO ADD this entry while testing with local repos.

Reply:
to test on local, you can add the [trusted=yes] in the sources list of apt.
e.g., if your packages are placed in mypkg, the entry in sources.list will be:
```deb [trusted=yes] file:/home/foglamp/mypkg ./```

Reply:
See my previous comments. I have already tried this option in sources.list and it is not working and that is why this --allow-unauthenticated option was added for local testing.